crown-court-proceeding/src/testSqsIntegration/java/integration/ProsecutionMessageBuilder.java

```diff
@@ -0,0 +1,214 @@
+package integration;
+
+public class ProsecutionMessageBuilder {
+
+
+    public static final String CONVICTED = "CONVICTED";
+    public static final String AQUITTED = "AQUITTED";
+    public static final String PART_CONVICTED = "PART CONVICTED";
+    public static final String GUILTY = "GUILTY";
+    public static final String NOT_GUILTY = "NOT_GUILTY";
+
+    public static String getSqsMessagePayload(Integer maatId) {
+        return """
+                {
+                   prosecutionCaseId : 998984a0-ae53-466c-9c13-e0c84c1fd581,
+                   defendantId: aa07e234-7e80-4be1-a076-5ab8a8f49df5,
+                   isConcluded: true,
+                   hearingIdWhereChangeOccurred : 908ad01e-5a38-4158-957a-0c1d1a783862,
+                    offenceSummary: [
+                        """ + getOffenceWithPleaAndVerdict(GUILTY, GUILTY) + """
```

Comment:
Instead of dynamically creating the JSON, can we create test data for each scenario and externalise it in the resources/data folder? Also, we can reuse the Crime Commons FileUtils to read the data from file as string Example: [FileUtilsTest.java](https://github.com/ministryofjustice/laa-crime-commons/blob/main/crime-commons-classes/src/test/java/uk/gov/justice/laa/crime/util/FileUtilsTest.java)